doc/glossary.rst

```diff
@@ -12,12 +12,13 @@
 	:ref:`BlueStore<rados_config_storage_devices_bluestore>`
                 OSD BlueStore is a storage back end used by OSD daemons, and
                 was designed specifically for use with Ceph. BlueStore was
-                introduced in the Ceph Kraken release. In the Ceph Luminous
-                release, BlueStore became Ceph's default storage back end,
-                supplanting FileStore. BlueStore stores objects directly on 
-                Ceph block devices without any file system interface. 
-                Since Luminous (12.2), BlueStore has been Ceph's default 
-                and recommended storage back end.
+                introduced in the Ceph Kraken release. In Ceph Luminous
+                release 12.2 BlueStore became Ceph's default storage back end,
+                supplanting FileStore. In the Reef release, FileStore was no
+                longer available as a storage backend.
+                
+                BlueStore stores objects directly on Ceph block devices without
+                any file system interface.  
```

Comment:
Suggest "BlueStore stores objects directly on block devices without
                a mounted file system. "

Reply:
To me a "Ceph block device" is an RBD volume, just sayin..